estimate_covid_infections_from_deaths_data.R

```diff
@@ -0,0 +1,149 @@
+# ============================================================================== #
+
+# Extract distribution the incubation period for COVID-19
+covid_incubation_dist <- epiparameter_db(
+  disease = "covid",
+  epi_name = "incubation",
+  single_epiparameter = TRUE
+)
+
+# Display the incubation period distribution information
+covid_incubation_dist
+
+# Extract parameters for EpiNow2 using LogNormal distribution
+incubation_params <- get_parameters(covid_incubation_dist)
+incubation_max_days <- round(quantile(covid_incubation_dist, 0.999))  # Upper 99.9% range needed for EpiNow2
+
+# Create a LogNormal object for the incubation period
+incubation_lognormal <- EpiNow2::LogNormal(
+  meanlog = incubation_params[["meanlog"]],
+  sdlog = incubation_params[["sdlog"]],
+  max = incubation_max_days
+)
+
+# Get the onset-to-death distribution
+onset_to_death_dist <- epiparameter_db(
+  disease = "covid",
+  epi_name = "onset to death",
+  single_epiparameter = TRUE
+)
+
+# Display the onset-to-death distribution information
+onset_to_death_dist
+
+# Extract parameters for EpiNow2 using LogNormal distribution
+onset_death_params <- get_parameters(onset_to_death_dist)
+onset_death_max_days <- round(quantile(onset_to_death_dist, 0.999))  # Upper 99.9% range
+
+# Create an EpiNow2 LogNormal object for the onset-to-death distribution
+onset_death_lognormal <- EpiNow2::LogNormal(
+  meanlog = onset_death_params[["meanlog"]],
+  sdlog = onset_death_params[["sdlog"]],
+  max = onset_death_max_days
+)
+
+# Use EpiNow2 to convolve the infection-to-onset and onset-to-death distributions
+infection_to_death_dist <- incubation_lognormal + onset_death_lognormal
+
+# Extract the serial interval distribution
+serial_interval_dist <- epiparameter_db(
+  disease = "covid",
+  epi_name = "serial",
+  single_epiparameter = TRUE
+)
+
+# Display the serial interval distribution information
+serial_interval_dist
+
+# Find the upper 99.9% range for the serial interval
+serial_interval_max_days <- round(quantile(serial_interval_dist, 0.999))
+
+# Extract parameters for the serial interval
+serial_interval_params <- get_parameters(serial_interval_dist)
+
+# Create a LogNormal object for the serial interval
+serial_interval_lognormal <- EpiNow2::LogNormal(
+  meanlog = serial_interval_params[["meanlog"]],
+  sdlog = serial_interval_params[["sdlog"]],
+  max = serial_interval_max_days
+)
+
+# ============================================================================== #
+# ESTIMATE INFECTIONS AND VISUALIZE RESULTS
+# ============================================================================== #
+
+# Estimate infections using the non-mechanistic model
+infection_estimates <- epinow(
+  data = uk_covid_deaths,  # Time series data
+  generation_time = generation_time_opts(serial_interval_lognormal),  # Generation time
+  delays = delay_opts(infection_to_death_dist),  # Delay from infection to death
+  rt = NULL  # Rt is not estimated
```

Comment:
should we add this additional argument, then?
```suggestion
  rt = NULL,  # Rt is not estimated
  gp = gp_opts(alpha = Normal(mean = 0,sd = 0.05)) # Gaussian Process lengthscale prior
```

Reply:
would you like to add a message to suggest that action, assuming the argument would be unknown to participants or possibly too hidden in the whole script?

Reply:
or provide two figures when showcasing the output before using the cloud

Reply:
See 23398464d12dee9b131bc1e4346293ac010f7133.